.postcssrc.yml

```diff
@@ -0,0 +1,3 @@
+plugins:
+  postcss-import: {}
+  postcss-cssnext: {}
```

Comment:
Do you familiar with this plugin? If you haven't ever used it I would recommend not to install it in order to keep focus on our default stack.